.github/workflows/build-arm-releases.yml

```diff
@@ -59,12 +59,8 @@ jobs:
           version: "21.7"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: actions-rust-lang/setup-rust-toolchain@v1
-        with:
-          rustflags: --allow warnings -C target-feature=+aes
-          components:
-            cargo
-            rustfmt
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup-rust
```

Comment:
Using git submodule is also a workaround  `https://github.com/apache/celeborn/pull/3431`

Reply:
do we need to switch to sub module？

Reply:
I think it's OK, it's easy to maintain and upgrade the version, WDYT?